### PR TITLE
snippets and settings for vscode

### PR DESCRIPTION
**Pull request by WalkerTexas-coder**

### CODE REVIEW

File: tools-and-resources/vscode-snippets-settings.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+## Snippets
+VS Code User Snippets v0.1
+
+Instructions
+1. Cmd + shift + “p”
+2. “Preferences: Configure User Snippets”
+3. Select “javascript.json”
+4. Copy and paste code below
+5. save file
```
Comment (sjproctor) [inline tools-and-resources/vscode-snippets-settings.md:11]: Prolly capitalize Save to be consistent.

File: tools-and-resources/vscode-snippets-settings.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+JavaScript:
+```json
+{
+	"Console Log": {
+		"prefix":"cl",
+		"body": "conslone.log($1)",
```
Comment (sjproctor) [inline tools-and-resources/vscode-snippets-settings.md:18]: Still have the typo here

File: tools-and-resources/vscode-snippets-settings.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+		"description": "Console Log"
+	},  
+	"For Loop": {
+		"prefix": "for", 
+		"body": [
+			"for( let i = 0; i < array.length; i++ ){",
+			"  const ${1:element} = array[i]",
+			"}"
+		],
```
Comment (sjproctor) [inline tools-and-resources/vscode-snippets-settings.md:27]: I think we decided on `for(let i = 0; i < array.length; i++) {"` - no spaces around the content in the parens but a space after the closing paren before the opening curly.

File: tools-and-resources/vscode-snippets-settings.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+  },
+  "Arrow Function": {
+    "prefix": "arwfn",
+    "body": ["const ${1:functionName} = ($2) => {", "  $3", "}"],
+    "description": "Arrow Function"
+  },
+  "Basic If Statement": {
+    "prefix": "if",
+    "body": ["if( ${1:condition} ) {", " $2 ", "}"],
+    "description": "If Statement"
```
Comment (sjproctor) [inline tools-and-resources/vscode-snippets-settings.md:43]: I can't tell for sure but make sure there are no spaces around the condition `if(${1:condition})`

**Review by sjproctor — CHANGES_REQUESTED** [review]
Couple notes for your review.

File: tools-and-resources/vscode-snippets-settings.md (hunk 2 above)
Comment (WalkerTexas-coder) [reply tools-and-resources/vscode-snippets-settings.md:18]: hhhhmm that's weird

File: tools-and-resources/vscode-snippets-settings.md (hunk 3 above)
Comment (sjproctor) [reply tools-and-resources/vscode-snippets-settings.md:28]: WOOO!

**Review by sjproctor — APPROVED** [review]
Great!

**Pull request merged**